src/Mount.cpp

```diff
@@ -748,31 +747,33 @@ bool Mount::connectToDriver( TMC2209Stepper* driver, const char *driverKind ) {
     _driverFocus->beginSerial(19200);
     _driverFocus->mstep_reg_select(true);
     _driverFocus->pdn_disable(true);    
-    bool _UART_Rx_connected = false;
     #if UART_CONNECTION_TEST_TXRX == 1
-        _UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
-        if (!_UART_Rx_connected) {
-            digitalWrite(FOCUS_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
-        }
+      bool UART_Rx_connected = false;
+      UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
+      if (!UART_Rx_connected) {
+          digitalWrite(FOCUS_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
+      }
     #endif
     _driverFocus->toff(0);
     #if USE_VREF == 0  //By default, Vref is ignored when using UART to specify rms current.
         _driverFocus->I_scale_analog(0);
```

Comment:
Don't use integer literals for boolean values

Reply:
Making me fix legacy code.... tsk tsk

Reply:
If no-one fixes it it'll stay legacy code :stuck_out_tongue_closed_eyes: